engine/src/configuration/configuration.h

```diff
@@ -372,10 +435,12 @@ class Configuration {
     vega_config::AudioConfig audio_config_;
     vega_config::CollisionHacks collision_hacks;
     vega_config::ComputerConfig computer_config_;
+    vega_config::EjectConfig eject_config_;
```

Comment:
I'd be in favor of dropping the `_` at the end....it'd make it a little easier to read

Reply:
Done.